protostuff-runtime/test-map-collection-register.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mvn -o -DforkMode=never \
+    -Dprotostuff.runtime.pojo_schema_on_collection_fields=true \
```

Comment:
Agreed.  Adding a way to construct an IdStrategy with these flags should do it (next pr).